hw/ip_templates/rv_core_ibex/rtl/rv_core_ibex.sv.tpl

```diff
@@ -713,13 +719,33 @@ module ${module_instance_name}
     .rst_ni,
     .tl_h_i      (tl_d_ibex2fifo),
     .tl_h_o      (tl_d_fifo2ibex),
-    .tl_d_o      (cored_tl_h_o),
-    .tl_d_i      (cored_tl_h_i),
+    .tl_d_o      (tl_d_fifo2gate),
+    .tl_d_i      (tl_d_gate2fifo),
     .spare_req_i (1'b0),
     .spare_req_o (),
     .spare_rsp_i (1'b0),
     .spare_rsp_o ());
 
+  // Gate any pending requests on escalation
+  tlul_lc_gate #(
+    .Outstanding(NumOutstandingReqs),
+    .NumGatesPerDirection(1),
+    // Don't return an error. Only interested in blocking a request at this stage
+    .ReturnBlankResp(1)
+  ) u_tlul_lc_gate_cored (
+    .clk_i,
+    .rst_ni,
+    .tl_h2d_i       (tl_d_fifo2gate),
+    .tl_d2h_o       (tl_d_gate2fifo),
+    .tl_h2d_o       (cored_tl_h_o),
+    .tl_d2h_i       (cored_tl_h_i),
+    .flush_req_i    (1'b0),
+    .flush_ack_o    (),
+    .resp_pending_o (),
+    .lc_en_i        (lc_cpu_en[0]),
```

Comment:
Just checking: do you intend this to use the same copy as the one from which we derive `fetch_enable`?

Reply:
I *think* so (which you can do by passing a `NumCopies` parameter to `prim_lc_sync`). I'm definitely not a security expert, though. It's probably worth running it past one as well (@vogelpi ?)